includes/TypesForMembers/MemberRotationalRestraint.js

```diff
@@ -0,0 +1,361 @@
+ */
+MemberRotationalRestraint.prototype.Discrete = function (no,
+    member_supports_no,
+    material_name,
+    section_name,
+    rotational_stiffness,
+    continuous_beam_effect,
+    section_deformation_cdb,
+    comment,
+    params) {
+    this.member_rotational_restraint = createMemberRotationalRestraint(no, member_supports_no, "DISCRETE", comment, params);
+    ASSERT(typeof material_name !== "undefined", "Sheeting material name must be defined");
+    ASSERT(typeof section_name !== "undefined", "Section name must be specified");
+    this.member_rotational_restraint.material_name = material_name;
+    this.member_rotational_restraint.section_name = section_name;
+    this.member_rotational_restraint.rotational_stiffness = GetMemberRotationalRestraintStiffnessType(rotational_stiffness);
+    this.member_rotational_restraint.continuous_beam_effect = GetMemberRotationalRestraintContinuousBeamEffectType(continuous_beam_effect);
+    if (typeof section_deformation_cdb === "undefined") {
+        section_deformation_cdb = true;
+    }
+    this.member_rotational_restraint.section_deformation_cdb = section_deformation_cdb;
+};
+
+/**
+ * Creates Manually Member Rotational Restraint
+ * @param {Number}  no                  Index of Member Rotational Restraint, can be undefined
+ * @param {Array}   member_supports_no  List of assigned member supports indexes, can be undefined
+ * @param {String}  comment             Comment, can be undefined 
+ * @param {Object}  params              Additional parameters, can be undefined
+ */
+MemberRotationalRestraint.prototype.Manually = function (no,
+    member_supports_no,
+    comment,
+    params) {
+    this.member_rotational_restraint = createMemberRotationalRestraint(no, member_supports_no, "MANUALLY", comment, params);
+};
+
+/**
+ * Sets parameters for Continuous Member Rotational Restraint
+ * @param {Number} modulus_of_elasticity        Modulus of elasticity, can be undefined (default value taken from selected sheeting material)
+ * @param {Number} sheeting_thickness           Sheeting thickness, can be undefined (default value taken from selected section)
+ * @param {Number} sheeting_moment_of_inertia   Sheeting moment of inertia, can be undefined (default value taken from selected section)
+ * @param {Number} sheeting_distance_of_ribs    Sheeting distance of ribs, can be undefined (default value taken from selected section)
+ * @param {Number} width_of_section_flange      Width of sheeting flange, can be undefined (default value taken from selected section)
+ * @param {Number} spring_stiffness             Spring stiffness, can be undefined (different depending on loading as default)
+ * @param {Number} beam_spacing                 Beam spacing, can be undefined (3 m as default)
+ */
+MemberRotationalRestraint.prototype.SetContinuousParameters = function (modulus_of_elasticity,
+    sheeting_thickness,
+    sheeting_moment_of_inertia,
+    sheeting_distance_of_ribs,
```

Comment:
This sis taken from sheeting name - parameter of sheeting - delete all params connected